libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/NotificationActionListener.kt

```diff
@@ -0,0 +1,5 @@
+package com.mapbox.navigation.base.trip
+
+interface NotificationActionListener {
```

Comment:
This doesn't need to be part of `libnavigation-base` and could be exposed in `libandroid-navigation` (core).

Reply:
Once we have the `core` back from the driver app to the SDK we can move it